<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QMainWindow" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>479</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="1">
     <widget class="QPushButton" name="ButtonSkobka2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #1E90FF;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #0000FF;
	border: 1px solid gray;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>)</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QLabel" name="Answer">
      <property name="styleSheet">
       <string notr="true">QLabel {
	background-color: #ffffff;
	border: 1px solid gray;
	color: #191970;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="BMod">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #FF8C00;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #FF6347;
	border: 1px solid gray;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>mod</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="Button5">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #3CB371;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #32CD32;
	border: 1px solid gray;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="Button8">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #3CB371;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #32CD32;
	border: 1px solid gray;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="3" column="5">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>X Max</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignHCenter</set>
      </property>
     </widget>
    </item>
    <item row="7" column="4">
     <widget class="QLineEdit" name="ymin">
      <property name="text">
       <string>-10</string>
      </property>
     </widget>
    </item>
    <item row="8" column="2">
     <widget class="QPushButton" name="BAtan">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #20B2AA;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #008B8B;
	border: 1px solid gray;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>atan</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QPushButton" name="BCos">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #20B2AA;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #008B8B;
	border: 1px solid gray;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>cos</string>
      </property>
     </widget>
    </item>
    <item row="4" column="5">
     <widget class="QLineEdit" name="xmax">
      <property name="text">
       <string>10</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QPushButton" name="ButtonEqual">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #008000;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #FFFF00;
	border: 1px solid gray;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="ButtonDot">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #3CB371;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #32CD32;
	border: 1px solid gray;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>.</string>
      </property>
     </widget>
    </item>
    <item row="8" column="3">
     <widget class="QPushButton" name="BLog">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #20B2AA;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #008B8B;
	border: 1px solid gray;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>log</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="Button3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #3CB371;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #32CD32;
	border: 1px solid gray;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QPushButton" name="Division">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #FF8C00;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #FF6347;
	border: 1px solid gray;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>/</string>
      </property>
     </widget>
    </item>
    <item row="8" column="4" colspan="2">
     <widget class="QPushButton" name="ButtonGraph">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #008000;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #FFFF00;
	border: 1px solid gray;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>Построить график</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>X = </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="Multi">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #FF8C00;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #FF6347;
	border: 1px solid gray;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>*</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QPushButton" name="Button0">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #3CB371;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #32CD32;
	border: 1px solid gray;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="ButtonSkobka1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #1E90FF;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #0000FF;
	border: 1px solid gray;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>(</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="Plus">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #FF8C00;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #FF6347;
	border: 1px solid gray;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="7" column="5">
     <widget class="QLineEdit" name="ymax">
      <property name="text">
       <string>10</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="Button2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #3CB371;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #32CD32;
	border: 1px solid gray;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="Button7">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #3CB371;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #32CD32;
	border: 1px solid gray;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="Button9">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #3CB371;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #32CD32;
	border: 1px solid gray;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="6" column="4">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Y Min</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignHCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QLineEdit" name="xmin">
      <property name="text">
       <string>-10</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QPushButton" name="BSqrt">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #20B2AA;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #008B8B;
	border: 1px solid gray;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>sqrt</string>
      </property>
     </widget>
    </item>
    <item row="6" column="5">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Y Max</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignHCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="Button6">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #3CB371;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #32CD32;
	border: 1px solid gray;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="Minus">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #FF8C00;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #FF6347;
	border: 1px solid gray;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="4">
     <widget class="QLineEdit" name="Display">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>PT Sans Caption</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
	background-color: #ffffff;
	border: 1px solid gray;
	color: #191970;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QLineEdit" name="textX">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="Button4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #3CB371;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #32CD32;
	border: 1px solid gray;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="QPushButton" name="BSqr">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #FF8C00;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #FF6347;
	border: 1px solid gray;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>^</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QPushButton" name="BAcos">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #20B2AA;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #008B8B;
	border: 1px solid gray;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>acos</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QPushButton" name="BTan">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #20B2AA;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #008B8B;
	border: 1px solid gray;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>tan</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QPushButton" name="BAsin">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #20B2AA;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #008B8B;
	border: 1px solid gray;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>asin</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4" colspan="2">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Avenir</family>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Область определения</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignHCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="Button1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #3CB371;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #32CD32;
	border: 1px solid gray;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QPushButton" name="BSin">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #20B2AA;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #008B8B;
	border: 1px solid gray;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>sin</string>
      </property>
     </widget>
    </item>
    <item row="7" column="3">
     <widget class="QPushButton" name="BLn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #20B2AA;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #008B8B;
	border: 1px solid gray;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>ln</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>X Min</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignHCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="4" colspan="2">
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>Avenir</family>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Область значения</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignHCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>764</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
